test/prototype_transforms_kernel_infos.py

```diff
@@ -104,23 +112,34 @@ def sample_inputs_horizontal_flip_mask():
 )
 
 
+def _get_resize_sizes(image_size):
+    height, width = image_size
+    yield height, width
+    yield int(height * 0.75), int(width * 1.25)
+
+
 def sample_inputs_resize_image_tensor():
     for image_loader, interpolation in itertools.product(
         make_image_loaders(dtypes=[torch.float32]),
         [
-            F.InterpolationMode.NEAREST,
```

Comment:
Why did you remove this mode ?

Reply:
Because the only branching is happening on `BICUBIC`

https://github.com/pytorch/vision/blob/c0911e31d367eca9600541bd634564c73753abbf/torchvision/transforms/functional_tensor.py#L466-L467

Note that we are talking about the sample inputs here. The full matrix with all interpolation modes is still there for the reference inputs. L127-L131

Reply:
Sorry, I do not understand anymore how it works :)

Reply:
As discussed offline this should be nearest and bicubic.